tracer/resources/args_parser.py

```diff
@@ -16,7 +16,7 @@
 # 02110-1301, USA.
 #
 
-import argparse
+import argparse, argcomplete
```

Comment:
Arch doesn't have it in the official repos for example. While users can get it from the AUR it makes sense for it to be an optional dependency there.